index.js

```diff
@@ -34,6 +33,35 @@ var pairSplitRegExp = /; */;
 
 var fieldContentRegExp = /^[\u0009\u0020-\u007e\u0080-\u00ff]+$/;
 
+/**
+ * Is the codepoint a space or a horizontal tab?
+ *
+ * @param {number} code
+ * @return {boolean}
+ */
+function isWhitespace(code) {
+  return code === 0x20 || code === 0x09;
+}
+
+/**
+ * Trims the string of leading and trailing whitespace
+ *
+ * @param {string} str
+ * @param {number} start
+ * @param {number} end
+ * @return {string}
+ */
+
+function trim(str, start, end) {
```

Comment:
Is there a module on npm we can use instead of maintaining this?

Reply:
Not as specialized as this. Most just call `str.replace(/^\s+|\s+$/g, '')` (which is much slower than calling `str.trim()`), or just mimick `str.trim` with `/[\s\uFEFF\u00A0]/`.

Reply:
Don't we want it to behave the same, or is this a semver major change or behavior?